src/Analyzers/CSharp/Tests/UseCollectionInitializer/UseCollectionInitializerTests_CollectionExpression.cs

```diff
@@ -1525,9 +1535,191 @@ class C
+                    [
+                        // Goo
+                        .. x,
+                        // Bar
+                        .. y
+                    ];
+                }
+            }
+            """);
+    }
+
+    [Fact]
+    public async Task TestTrivia3()
+    {
+        await TestInRegularAndScriptAsync(
+            """
+            using System.Collections.Generic;
+            class C
+            {
+                void M(int[] x, int[] y)
+                {
+                    List<int> c = [|new|] List<int>();
+                    // Goo
+                    // Bar
+                    [|foreach (var v in |]x)
+                        c.Add(v);
+
+                    // Baz
+                    // Quux
+                    [|foreach (var v in |]y)
+                        c.Add(v);
+                }
+            }
+            """,
+            """
+            using System.Collections.Generic;
+            class C
+            {
+                void M(int[] x, int[] y)
+                {
+                    List<int> c =
+                    [
+                        // Goo
+                        // Bar
+                        .. x,
+                        // Baz
+                        // Quux
+                        .. y
+                    ];
+                }
+            }
+            """);
+    }
+
+    [Fact]
+    public async Task TestTrivia4()
+    {
+        await TestInRegularAndScriptAsync(
+            """
+            using System.Collections.Generic;
+            class C
+            {
+                void M(bool b1, bool b2)
+                {
+                    List<int> c = [|new|] List<int>();
+                    // Goo
+                    if (b1)
+                        c.Add(0);
+                    else
+                        c.Add(1);
+
+                    // Bar
+                    if (b2)
+                    {
+                        c.Add(2);
+                    }
+                    else
+                    {
+                        c.Add(3);
+                    }
+                }
+            }
+            """,
+            """
+            using System.Collections.Generic;
+            class C
+            {
+                void M(bool b1, bool b2)
+                {
+                    List<int> c =
+                    [
+                        // Goo
+                        b1 ? 0 : 1,
+                        // Bar
+                        b2 ? 2 : 3
+                    ];
+                }
+            }
+            """);
+    }
+
+    [Fact]
+    public async Task TestTrivia5()
+    {
+        await TestInRegularAndScriptAsync(
+            """
+            using System.Collections.Generic;
+            class C
+            {
+                void M(bool b1, bool b2)
+                {
+                    List<int> c = [|new|] List<int>();
+                    // Goo
+                    // Bar
+                    if (b1)
```

Comment:
Can you add a test like so
```suggestion
                    if (b1) // Comment
```

Reply:
yes.  i'll add in followup pr.  right now that comment will be removed.